src/main/java/com/epam/ta/reportportal/commons/SendCase.java

```diff
@@ -0,0 +1,39 @@
+package com.epam.ta.reportportal.commons;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+/**
+ * Email notification cases enumerator for project settings
+ *
+ * @author Andrei_Ramanchuk
+ */
+public enum SendCase {
+
+	//@formatter:off
+	ALWAYS("always"),
+	FAILED("failed"),
+	TO_INVESTIGATE("to_investigate"),
```

Comment:
let's make it camel case as it is used by ui

Reply:
done